core/src/test/java/org/infinispan/stream/stress/DistributedStreamRehashStressTest.java

```diff
@@ -44,7 +44,7 @@
  * @author wburns
  * @since 8.0
  */
-@Test(groups = "stress", testName = "stream.stress.DistributedStreamRehashStressTest")
+@Test(groups = "stress", testName = "stream.stress.DistributedStreamRehashStressTest", enabled = false)
```

Comment:
Whoops it wasn't meant to be disabled.  I only did that so I could run the entire package more easily.  Thanks :)